src/widgets/settingspages/GeneralPage.cpp

```diff
@@ -625,11 +634,16 @@ void GeneralPage::initLayout(GeneralPageView &layout)
     layout.addCheckbox("Show parted users (< 1000 chatters)", s.showParts);
     layout.addCheckbox("Automatically close user popup when it loses focus",
                        s.autoCloseUserPopup);
-    layout.addCheckbox("Lowercase domains (anti-phishing)", s.lowercaseDomains);
+    layout.addCheckbox("Lowercase domains (anti-phishing)", s.lowercaseDomains,
+                       false,
+                       "Make all clickable links lowercase to deter "
+                       "phishing attempts.");
     layout.addCheckbox("Bold @usernames", s.boldUsernames);
     layout.addCheckbox("Color @usernames", s.colorUsernames);
     layout.addCheckbox("Try to find usernames without @ prefix",
-                       s.findAllUsernames);
+                       s.findAllUsernames, false,
+                       "Attempt to autocomplete usernames "
+                       "without the @ prefix.");
```

Comment:
This isn't related to autocompletion. This is finding usernames in chat.
```suggestion
                       "Find mentions of users in chat "
                       "without the @ prefix.");
```
(will need reformat)